### PR TITLE
Test target size logic in upgrade test

### PR DESCRIPTION
**Pull request by sisuresh**
# Description

1. Hit target size in upgrade test.
2. Add safety check in utils.
3. Increase fees to handle hitting target size.

# Checklist
- [ ] Reviewed the [contributing](https://github.com/stellar/stellar-core/blob/master/CONTRIBUTING.md#submitting-changes) document
- [ ] Rebased on top of master (no merge commits)
- [ ] Ran `clang-format` v8.0.0 (via `make format` or the Visual Studio extension)
- [ ] Compiles
- [ ] Ran all tests
- [ ] If change impacts performance, include supporting evidence per the [performance document](https://github.com/stellar/stellar-core/blob/master/performance-eval/performance-eval.md)

### CODE REVIEW

File: src/main/SettingsUpgradeUtils.cpp [hunk 1]
```diff
@@ -133,23 +133,50 @@ getCreateTx(PublicKey const& publicKey, LedgerKey const& contractCodeLedgerKey,
...
     tx.ext.v(1);
     tx.ext.sorobanData().resources = uploadResources;
-    tx.ext.sorobanData().resourceFee = 1'000'000;
+    tx.ext.sorobanData().resourceFee = 3'000'000;
 
     return {txEnv, contractSourceRefLedgerKey, contractID};
 }
 
+void
+validateConfigUpgradeSet(ConfigUpgradeSet const& upgradeSet)
```
Comment (dmkozh) [inline src/main/SettingsUpgradeUtils.cpp:142]: Can we quickly cover this check triggering in some test?

Comment (sisuresh) [reply src/main/SettingsUpgradeUtils.cpp:142]: Done

**Comment by dmkozh** [comment]
r+ 6cdf449

**Comment by dmkozh** [comment]
r+ 249e42f

**Pull request merged**